lib/core/test/unit/Cardano/Wallet/DB/Arbitrary.hs

```diff
@@ -413,6 +414,14 @@ instance Arbitrary UTxO where
             <*> vector n
         return $ UTxO $ Map.fromList u
 
+
+instance Arbitrary a => Arbitrary (Range a) where
+    arbitrary = Range <$> arbitrary <*> arbitrary
+
+    shrink (Range from to) =
+        [Range from' to | from' <- shrink from]
+        ++ [Range from to' | to' <- shrink to]
```

Comment:
Are we sure that `to'` is necessarily bigger than `from` here ? For `from'`, it should shrink to a value smaller than `from` so there's no obvious problem, but for `to'` :shrug: